app/src/main/java/com/neongarage/Fragments/CountInFragment.java

```diff
@@ -136,29 +142,62 @@ public void handleSubmitClick(View v) {
         double dimeTotal = dimes.getTotal();
         double quarterTotal = quarters.getTotal();
         double coinValue = nickleTotal + quarterTotal + dimeTotal;
-        coinTotal.setText("Nickels: " + df.format(nickleTotal) + "\nDimes: " + df.format(dimeTotal) + "\nQuarters: " + df.format(quarterTotal));
+        coins.append("Nickels: ");
+        coins.append(df.format(nickleTotal));
```

Comment:
you can do this all at once.
coins.append(df.format(nickleTotal)).append("\nDimes: ").append(df.format(dimeTotal)) etc.